### PR TITLE
Curriculum Launch 2023 - Homepage skinny banner

### PR DESCRIPTION
**Pull request by kelbyhawn**
Will replace the Curriculum Launch hero banner with a skinny banner on the homepage when we make the AI PL hero banner live.

- Uses the `curriculum-launch-skinny-banner` DCDO flag
  -  The `curriculum-launch-hero-banner` flag needs to be set to `false`
- Moved the TeachAI homepage skinny banner to https://code.org/ai
  - Renamed the files and classes to remove the `homepage` piece

**Jira ticket:** [ACQ-731](https://codedotorg.atlassian.net/browse/ACQ-731)

**Related PR:** 
- https://github.com/code-dot-org/code-dot-org/pull/52943

----

Desktop

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/teach_ai_skinny_banner.haml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-= inline_css 'generated/homepage-teach-ai-banner.css'
+= inline_css 'generated/teach-ai-skinny-banner.css'
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/views/teach_ai_skinny_banner.haml:1]: Is this also supposed to be behind the DCDO flag?

**Review by TurnerRiley — APPROVED** [review]
Looks great! Just had one quick question

File: pegasus/sites.v3/code.org/views/teach_ai_skinny_banner.haml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/views/teach_ai_skinny_banner.haml:1]: Nope! It used to be behind a flag but we removed it [here](https://github.com/code-dot-org/code-dot-org/pull/52464/files#diff-9d6502fdbc36cecfe029344c72579bfa407731327c21d8f5da94dee14289788b). With the move to the code.org/ai page it's ok to have it evergreen for now.

Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/views/teach_ai_skinny_banner.haml:1]: Gotcha! Sounds good :)

**Pull request merged**